app/controllers/needs_controller.rb

```diff
@@ -1,9 +1,9 @@
 class NeedsController < ApplicationController
   before_action :set_headers
-  before_action :set_need, only: [:show, :edit, :update, :destroy]
+  before_action :set_need, only: [:show, :edit, :update, :destroy, :archive]
 
   def index
-    @needs = Need.all
+    @needs = Need.where(active: !false)
```

Comment:
Should probably have this check on the api controllers too. 

might be better to have a default_scope for need / shelter to be `active: true`

Reply:
Not a fan of default scopes.  They always seem to cause problems.

Adding an active scope sounds like a great idea though.

Reply:
@ironcoconut I mean true, but this is precisely a time when a default scope would have solved this and now we're going to have to remember always and forever to `Need.active`

Reply:
Except when we really do want all records.

Idk. This does seem like a compelling use case since we are hiding deleted records from the default query.

I'm sold. Default scope works for me.